### DISCUSSION TITLE
Companies and projects using SCI

### DISCUSSION THREAD

**Discussion by borkdude** [borkdude/sci#662] [General]
## This is a follow up to [this](https://github.com/babashka/babashka/discussions/1026) thread.

Please share if your company or project is using SCI. You're welcome to provide a little bit of detail about how you are using it well. No strings attached.

This has several purposes:

- Your company might be mentioned in a sheet of a presentation about SCI. A link to a logo would be helpful.
- This issue would be a good way to notify you of any breaking changes in SCI (which we will try to prevent).
- Your company is listed on [this](https://github.com/babashka/sci/blob/master/doc/companies.md) page (TODO, see [here](https://github.com/babashka/babashka/blob/master/doc/companies.md) for a similar page about babashka)).
- It would help me a lot to understand who is using and how this project is being used.

If you would like to sponsor SCI and related projects, that'd be great as well:

https://opencollective.com/babashka
https://github.com/sponsors/borkdude

I'm also available for [commercial support](https://blog.michielborkent.nl/oss-highlights-nov-dec-2021.html#enterprise-support) if you need that.

Similar discussions for:

- [babashka](https://github.com/babashka/babashka/discussions/1026)
- [clj-kondo](https://github.com/clj-kondo/clj-kondo/discussions/1397)
- [nbb](https://github.com/borkdude/nbb/discussions/91)

**Comment by borkdude** [comment]
Portal (@djblue):

> Hi! Yeah, Portal uses it to allows users add [custom viewers](https://github.com/djblue/portal/blob/master/src/portal/ui/api.cljs#L6) and [script the client](https://clojurians.slack.com/archives/C0185BFLLSE/p1647891314417949). They can call portal.api/eval-str which gets sent to the ui and interpreted via sci which has a bunch of the [portal ui namespaces exposed](https://github.com/djblue/portal/blob/master/src/portal/ui/sci/libs.cljs). I don't have docs outside of the api documentation, I do think adding portal to the readme is a good idea

**Comment by den1k** [comment]
https://github.com/lumberdev/tesserae uses SCI!

**Reply by borkdude** [reply]
Awesome!

**Comment by jhandke** [comment]
As a part of my bachelor thesis I created a visual editor / programming environment for simple Clojure programs. It is intended for the first few weeks of the functional programming course at my university.
The project uses SCI to evaluate the generated code.

You can find everything over at Codeberg:
https://codeberg.org/jhandke/ClojureBlocks

**Reply by borkdude** [reply]
Very cool, thanks for sharing

**Reply by jhandke** [reply]
I moved the repository to Github: https://github.com/jhandke/ClojureBlocks

**Comment by kloimhardt** [comment]
Exited to discover that in '23 we have two Clojure visual programming environments using SCI: ClojureBlocks and https://github.com/kloimhardt/clj-tiles

**Comment by ingydotnet** [comment]
https://github.com/yaml/yamlscript uses SCI. 

https://yamlscript.org/

**Reply by borkdude** [reply]
Awesome!
